src/data/lib/prim/Agda/Builtin/Reflection/External.agda

```diff
@@ -0,0 +1,14 @@
+{-# OPTIONS --without-K --safe --no-sized-types --no-guardedness --no-subtyping #-}
```

Comment:
You'll need to add this file to `builtinModulesWithSafePostulates` in `src/full/Agda/Interaction/Options/Lenses.hs`.

Reply:
I've added `Agda.Builtin.Reflection.External` to this `builtinModulesWithSafePostulates`.